test_regress/t/t_vl_assign_sbw.cpp

```diff
@@ -0,0 +1,29 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+// This file ONLY is placed under the Creative Commons Public Domain, for
+// any use, without warranty, 2024 by Antmicro.
+// SPDX-License-Identifier: CC0-1.0
+
+#include VM_PREFIX_INCLUDE
+#include <systemc.h>
+
+#include <sysc/kernel/sc_simcontext.h>
+
+int sc_main(int argc, char* argv[]) {
+    using namespace sc_core;
+
+    VM_PREFIX* tb = new VM_PREFIX{"t"};
+    constexpr int val = 1;
+    sc_signal<sc_biguint<256>> SC_NAMED(in, val);
+    sc_signal<sc_biguint<256>> SC_NAMED(out);
+
+    tb->in(in);
+    tb->out(out);
+
+    sc_start(1, SC_NS);
+
```

Comment:
Can you please update the test so that it fails without your patch, and passes with it, thanks.

Reply:
While testing on the newest master without the fix, the test fails for the newest SystemC library. 
Shall I include any stacktrace of failed test output?

Reply:
Oh, ok, I guess just test the value really is passed across then, and maybe is zero(), which you mentioned, works.

Reply:
Sure, I'll modify the test case to include iszero() and pass-across checks.